kvm-simple-init

```diff
@@ -185,6 +185,8 @@ action="$1"
 retcode=0
 
 for vmname in $vmnames; do
+	vmname=${vmname%.conf}
```

Comment:
I just tested it, without that line the vm_start, vm_stop and vm_status scripts look for vmname.conf.conf, which isn't intended behaviour.